bundles/org.openhab.binding.gpio/src/main/java/org/openhab/binding/gpio/internal/handler/PigpioDigitalOutputHandler.java

```diff
@@ -33,16 +38,22 @@
  *
  * @author Nils Bauer - Initial contribution
  * @author Jan N. Klug - Channel redesign
+ * @author Jeremy Rumpf - Refactored for network disruptions
  */
 @NonNullByDefault
 public class PigpioDigitalOutputHandler implements ChannelHandler {
-
-    /** The logger. */
     private final Logger logger = LoggerFactory.getLogger(PigpioDigitalOutputHandler.class);
 
     private final GPIOOutputConfiguration configuration;
-    private final GPIO gpio;
-    private final Consumer<State> updateStatus;
+    private final ScheduledExecutorService scheduler;
+    private final Integer gpioId;
+    private Integer pulseTimeout = -1;
+    @Nullable
+    private String pulseCommand = "";
```

Comment:
```suggestion
    private @Nullable String pulseCommand = "";
```
Seen here and on secveral other places, i think inline annotation would be better